__tests__/app.test.js

```diff
@@ -106,6 +106,24 @@ describe("/api/articles", () => {
         expect(response.body.msg).toBe("incorrect input");
       });
   });
+  it("200: Will return all articles when inputting the topic", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        articles.forEach((article) => {
+          expect(article.topic).toBe("cats");
+        });
+      });
+  });
```

Comment:
I really like how clean this test it! You're not checking the other properties because they are already being covered by the tests above, this is great TDD 🏆